### PR TITLE
f/schools-table

### PR DESCRIPTION
**Pull request by whittlbc**
adding School table and getting manage.py to actually run

### CODE REVIEW

**Comment by whittlbc** [comment]
@atn34 having issues with migrating on top of the other migration already in master...had to blow away the migrations and start from scratch. Was due to our recent change to `id` from `user_id`. Unsure how you wanna go about fixing.

**Comment by atn34** [comment]
Yea I'm not really sure how to use Flask-Migrate yet. I think some of the time we'll have to tweak the autogenerated migration scripts

**Comment by whittlbc** [comment]
Do you wanna just gitignore the migrations directory for now? For the milestone at least?

**Comment by atn34** [comment]
gitignore-ing the migrations directory for now is fine with me

**Comment by whittlbc** [comment]
Okay cool. Will do that and fix tests

File: src/models.py [hunk 1]
```diff
@@ -1,39 +1,62 @@
...
-
-    def __repr__(self):
-        return '<User %r, id=%d>' % (self.email, self.id)
 
-class Token(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    user_id = db.Column(db.Integer, db.ForeignKey('user.id'))
-    secret = db.Column(db.String(64))
-    user = db.relationship('User', backref='tokens')
+class User(db.Model):
```
Comment (atn34) [inline src/models.py:13]: I'll switch to 2 spaces for the api as well

File: src/models.py [hunk 2]
```diff
@@ -1,39 +1,62 @@
...
+  is_destroyed = db.Column(db.Boolean(), default=False)
+  created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow)
+
+  def __init__(self, name, domains):
+    self.name = name
+    self.slug = slugify(name, separator='-', to_lower=True)
+    self.domains = domains
+
+  def __repr__(self):
+    return '<School id={}, name={}, slug={}, domains={}>'.format(self.id, self.name, self.slug, self.domains)
```
Comment (atn34) [inline src/models.py]: Nit: wrap this line?

**Review by atn34 — APPROVED** [review]
LGTM

**Pull request merged**